projects/storyteller/webapp/src/app/core/services/firebase.service.ts

```diff
@@ -25,53 +25,42 @@ export class FirebaseService {
       this.angularFireAuth.authState.subscribe(userData => {
       this.isOnline = !!userData;
       });
-      this.protobin = this.db.collection(`/storyteller/data/user/valerii.fedorenko.ua@gmail.com/story`);
-      this.protobin_screenshots = this.db.collection('/storyteller/data/user/valerii.fedorenko.ua@gmail.com/screenshot');
+      this.stories =
+      this.db.collection(`/storyteller/data/user/valerii.fedorenko.ua@gmail.com/story`);
+      this.screenshots =
+      this.db.collection('/storyteller/data/user/valerii.fedorenko.ua@gmail.com/screenshot');
   }
-
-  getstorylistAll(): Observable<StoryList[]> {
-    return this.protobin.snapshotChanges().pipe(
+  getStorylistAll(): Observable<StoryList[]> {
+    return this.stories.snapshotChanges().pipe(
         map(action => action.map(a => {
           const firebaseElement = a.payload.doc.data() as FirebaseElement;
 
           if (firebaseElement === undefined) {
             // Element not found
             return;
           }
-          return this.convertFirebaseElementToStory(firebaseElement);
+          return StoryList.deserializeBinary(this.getBinary(firebaseElement));
         })));
   }
-  getscreenshotAll(): Observable<Screenshot[]> {
-    return this.protobin_screenshots.snapshotChanges().pipe(
+  getScreenshotAll(): Observable<Screenshot[]> {
+    return this.screenshots.snapshotChanges().pipe(
         map(action => action.map(a => {
           const firebaseElement = a.payload.doc.data() as FirebaseElement;
 
           if (firebaseElement === undefined) {
             // Element not found
             return;
           }
-          return this.convertFirebaseElementToScreenshot(firebaseElement);
+          return Screenshot.deserializeBinary(this.getBinary(firebaseElement));
         })));
   }
-  private convertFirebaseElementToScreenshot(firebaseElement: FirebaseElement): Screenshot {
-    // Convert firebaseElement to binary
-    const binary: Uint8Array = this.encodingService
-      .decodeBase64StringToUint8Array(firebaseElement.proto);
-
-    // Convert binary to Screenshot
-    const screenshot: Screenshot = Screenshot.deserializeBinary(binary);
-    return screenshot;
-  }
-  private convertFirebaseElementToStory(firebaseElement: FirebaseElement): StoryList {
-
-    // Convert firebaseElement to binary
-    const binary: Uint8Array = this.encodingService
-      .decodeBase64StringToUint8Array(firebaseElement.proto);
-
-    // Convert binary to book
-    const storylist: StoryList = StoryList.deserializeBinary(binary);
-    return storylist;
-  }
+ // Converts firebaseElement to binary
+ private getBinary(firebaseElement: FirebaseElement): Uint8Array {
+  const binary: Uint8Array = this.encodingService.decodeBase64StringToUint8Array(
+    firebaseElement.proto
+  );
+  return binary;
```

Comment:
Please try to use code formatter :smile: 
It's `Shift`+`Ctrl`+`I` in VS code
`Ctrl`+`T`+`F` in Sublime